.github/workflows/publish-dev.yaml

```diff
@@ -79,6 +45,7 @@ jobs:
         shell: bash
 
       - name: Build and Publish package to TestPyPI
+        if: ${{ success() }}
```

Comment:
Why do we need this? I thought this wouldn't be executed, if a previous job failed.

Reply:
Okay according to the [Docs](https://docs.github.com/en/actions/learn-github-actions/expressions#status-check-functions):
```
A default status check of success() is applied unless you include one of these functions
```